src/atlas/CMakeLists.txt

```
@@ -974,7 +977,7 @@ ecbuild_add_library( TARGET atlas
     $<${atlas_HAVE_EIGEN}:Eigen3::Eigen>
     $<${atlas_HAVE_OMP_CXX}:OpenMP::OpenMP_CXX>
     $<${atlas_HAVE_GRIDTOOLS_STORAGE}:GridTools::gridtools>
-    $<$<AND:${atlas_HAVE_CUDA},${atlas_HAVE_NATIVE_STORAGE}>:CUDA::cudart>
+    $<${atlas_HAVE_NATIVE_STORAGE}:CUDA::cudart>
```

```suggestion
    $<${ATLAS_NATIVE_STORAGE_BACKEND_CUDA}:CUDA::cudart>
```
because this should only be the case when CUDA is enabled.